third_party/move/move-bytecode-verifier/src/verifier.rs

```diff
@@ -63,10 +63,21 @@ pub fn verify_module_with_config_for_test(
     name: &str,
     config: &VerifierConfig,
     module: &CompiledModule,
+) -> VMResult<()> {
+    verify_module_with_config_for_test_with_version(name, config, module, None)
+}
+
+pub fn verify_module_with_config_for_test_with_version(
```

Comment:
Probably fine for this PR, but this is quite a bad pattern I think:
1) pub functions for tests can be just `cfg(any(test, features = "testing"))` to avoid misuse (though name is descriptive so it's okay).
2) there are 2 functions to verify the module. I would prefer to see 1 where the caller provides the config, otherwise these defaults are really error-prone.

Reply:
Yes, not part of this PR, which should be focused and minimal.